CodeGen/docker/gaudi/docker_compose.yaml

```diff
@@ -72,6 +72,19 @@ services:
     ipc: host
     restart: always
 
+  codegen-gaudi-react-ui-server:
+    image: opea/codegen-react-ui:latest
+    container_name: codegen-gaudi-react-ui-server
+    depends_on:
+      - codegen-gaudi-backend-server
+    ports:
+      - "5174:80"
+    build:
```

Comment:
suggest to remove the build part.

Reply:
found some random issues in CI, 
![Uploading image.png…]()

Reply:
build part is need as we cannot inject environmental values to react once the image is built.
Can't the see the image that you have shared.